src/rviz/default_plugin/tf_display.cpp

```diff
@@ -178,9 +179,13 @@ TFDisplay::TFDisplay() : Display(), update_timer_(0.0f), changing_single_frame_e
                                            "Whether or not arrows from child to parent should be shown.",
                                            this, SLOT(updateShowArrows()));
 
-  scale_property_ =
+  scale_property_ = 
       new FloatProperty("Marker Scale", 1, "Scaling factor for all names, axes and arrows.", this);
+  
+  filter_property_ = new StringProperty("Filter", ".*", "Regex filter", this);
 
+  filter_blacklist_property_ = new StringProperty("Filter (blacklist)", "", "Regex filter", this);
```

Comment:
I suggest moving these properties as children of `frames_category_`. This makes it clear, where the filters are applied to. However, it makes the property harder to discover. What do you think?
Please rename "Filter" -> "Filter (whitelist)".